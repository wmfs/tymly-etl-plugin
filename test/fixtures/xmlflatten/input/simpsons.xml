<Simpsons>
    <Episode>
        <Title xml:lang="en">Cape Feare</Title>
        <People>
            <Person>
                <Name>Maggie</Name>
                <Age>1</Age>
                <Siblings>
                    <Brother>Bart</Brother>
                    <Sister>Lisa</Sister>
                </Siblings>
            </Person>
            <Person>
                <Name>Lisa</Name>
                <Age>8</Age>
                <Siblings>
                    <Brother>Bart</Brother>
                    <Sister>Maggie</Sister>
                </Siblings>
            </Person>
            <Person>
                <Name>Bart</Name>
                <Age>10</Age>
                <Siblings>
                    <Sister>Lisa</Sister>
                    <Brother>Maggie (wait! what?)</Brother>
                </Siblings>
            </Person>
            <Person>
                <Name>Marge</Name>
                <Age>36</Age>
            </Person>
            <Person>
                <Name>Sideshow Bob</Name>
            </Person>
        </People>
    </Episode>
    <Episode>
        <Title xml:lang="en">Homer Loves Flanders</Title>
        <People>
            <Person>
                <Name>Homer</Name>
                <Age>39</Age>
            </Person>
            <Person>
                <Name>Ned Flanders</Name>
                <Age>60</Age>
            </Person>
        </People>
    </Episode>
</Simpsons>
