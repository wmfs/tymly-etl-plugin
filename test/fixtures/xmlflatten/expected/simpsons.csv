"title","name","child","age"
"Cape Feare","Maggie","yes",
"Cape Feare","Lisa","yes",
"Cape Feare","Bart","yes",
"Cape Feare","Marge",,36
"Cape Feare","Sideshow Bob",,
"Homer Loves Flanders","Homer",,39
"Homer Loves Flanders","Ned Flanders",,60
